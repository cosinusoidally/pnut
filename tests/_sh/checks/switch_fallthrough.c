// expect_comp_failure
void main() {
  switch (1) {
    case 1:
      putchar('a');
    case 2:
      putchar('b');
      break;
  }
}
