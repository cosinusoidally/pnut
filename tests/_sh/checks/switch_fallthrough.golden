tests/_sh/checks/switch_fallthrough.c:11:0  case/default must be at the beginning of a switch conditional block
